dev/archery/archery/crossbow/core.py

```diff
@@ -1357,9 +1351,9 @@ def validate(self):
                 Task(task_name, **task)
             except Exception as e:
                 raise CrossbowError(
-                    'Unable to construct a task object from the '
-                    'definition  of task `{}`. The original error message '
-                    'is: `{}`'.format(task_name, str(e))
+                    f"Unable to construct a task object from the "
```

Comment:
`f` is not needed here

Reply:
Resolved in [2c7c552](https://github.com/apache/arrow/pull/45629/commits/2c7c552cf2119490867016bdce97578af1f1481b).